pnpm-workspace.yaml

```diff
@@ -1,3 +1,4 @@
 packages:
   - packages/*
   - packages/components-react/*
+  - packages/components-css/*
```

Comment:
Deze moet voor `changeset` ook aan de workspaces van `package.json` worden toegevoegd.

Reply:
We kijken even of dit nodig is en zo ja, dan voegen we het toe in een vervolg PR.